scripts/zones/Misareaux_Coast/npcs/Spatial_Displacement.lua

```diff
@@ -14,7 +14,7 @@ function onTrigger(player,npc)
     if (player:hasCompletedMission(COP,dsp.mission.id.cop.SHELTERING_DOUBT)) then
         player:startEvent(551); -- Access to Sites A & B
     elseif (player:getCurrentMission(COP) == dsp.mission.id.cop.ANCIENT_VOWS and player:getCharVar("PromathiaStatus") == 1) then
-        player:startEvent(8);
+        player:setPos(732.55,-32.5,-506.544,90,30); -- Go to Riverne #A01 {R}
```

Comment:
while the cs event that was here may have been incorrect, I am almost certain retail does indeed have an event here that does this move on its event finish, before the event that happens after the zone change, even if all that event does is play an animation of you being sucked through the portal.

This can be confirmed (or proven wrong) via a retail capture of the mission here.

Reply:
Noted: I'll addthe event for the option window to travel to RA01 and move stuff back to oneventfinish.

Reply:
Just watched video there is indeed an option to enter the site. Will add it back and test it and send it up when ready.